src/provider/amino/types/admin.ts

```diff
@@ -201,6 +241,14 @@ const AdminAmino: TypeUtils.SimpleMap<AminoConverter> = {
   [CarbonTx.Types.MsgRemoveMinGasPrice]: generateAminoType(MsgRemoveMinGasPrice),
   [CarbonTx.Types.MsgCreateValidator]: generateAminoType(MsgCreateValidator),
   [CarbonTx.Types.MsgEditValidator]: generateAminoType(MsgEditValidator),
+  [CarbonTx.Types.MsgAddRateStrategy]: generateAminoType(MsgAddRateStrategy),
+  [CarbonTx.Types.MsgUpdateRateStrategy]: generateAminoType(MsgUpdateRateStrategy),
+  [CarbonTx.Types.MsgRemoveRateStrategy]: generateAminoType(MsgRemoveRateStrategy),
+  [CarbonTx.Types.MsgAddAsset]: generateAminoType(MsgAddAsset),
+  [CarbonTx.Types.MsgUpdateAsset]: generateAminoType(MsgUpdateAsset),
+  [CarbonTx.Types.MsgSetLiquidationFee]: generateAminoType(MsgSetLiquidationFee),
+  [CarbonTx.Types.MsgSetInterestFee]: generateAminoType(MsgSetInterestFee),
+  [CarbonTx.Types.MsgSetStableCoinInterestRate]: generateAminoType(MsgSetStableCoinInterestRate),
 };
 
 export default AdminAmino;
```

Comment:
Backend updated codecs, may need to generate codecs + check if these are up-to-date